### ISSUE TITLE
mount-point in yang

### ISSUE THREAD

**Issue by mdoshi-com** [sysrepo/sysrepo#2174]
Hi,
Need some help with yangs having mount-point.
ex
```
augment "/agg:aggregated-o-ru/agg:aggregation" {
    container udp-echo-model {
      yangmnt:mount-point "udp-echo-root" {
        description
          "Root for o-ran-udp-echo.yang model supported per O-RU instance";
      }

```

What would be the xpath for attributes of o-ran-udp-echo.yang which are mounted here?
Is there something we have to do in ietf-yang-schema-mount.yang before mount-points can be accessed.

Regards,
Meghna

**Comment by michalvasko** [comment]
Hi,
schema mount is not supported so nothing will help you in this case, see https://github.com/CESNET/libyang/issues/1225.

**Comment by mdoshi-com** [comment]
Hi Michal,
Ok, so just confirming, without YANG Schema Mount support, mount-point wont work. Is this understanding correct?

Regards,
Meghna

**Comment by michalvasko** [comment]
Yes, the `mount-point` extension is defined in the `ietf-yang-schema-mount` module, this extension is not currently supported.

**Comment by safwan2m** [comment]
Hi @michalvasko, is this supported currently? Thanks.

**Comment by michalvasko** [comment]
Yes, it is and mentioned in the [docs](https://netopeer.liberouter.org/doc/sysrepo/devel/html/index.html#schema_mount).
